### PR TITLE
Lastline - improvements: add outputs, support array input, detonation playbooks, etc

### PR DESCRIPTION
**Pull request by bakatzir**
fixes: https://github.com/demisto/etc/issues/9298

api docs: https://analysis.lastline.com/analyst-api-docs/html/api.html#api.analysis.submit_file

~~add outputs~~
add infofile in outputs - only returns json.
~~support array input for pooling command~~
~~and detonate-file, detonate-url playbook~~
~~add test playbook~~
~~add documentation~~
~~pushtoportal - not optional~~ https://user.lastline.com/papi-doc/api/html/analysis/overview.html#analysis.submit_file

### CODE REVIEW

File: Playbooks/playbook-Detonate_File_-_Lastline.yml [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+    note: false
+  "1":
+    id: "1"
+    taskid: f142846a-82fb-4224-8c83-0debc6838bf1
+    type: regular
+    task:
+      id: f142846a-82fb-4224-8c83-0debc6838bf1
+      version: -1
+      name: Lastline Upload File
+      description: Uploads the submission to McAfee ATD.
```
Comment (BenJoParadise) [inline Playbooks/playbook-Detonate_File_-_Lastline.yml]: Please make sure descriptions are changed to reflect this integration.

File: Playbooks/playbook-Detonate_URL_-_Lastline.yml [hunk 2]
```diff
@@ -0,0 +1,341 @@
...
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 275,
+          "y": 50
+        }
+      }
+    note: false
+  "1":
```
Comment (BenJoParadise) [inline Playbooks/playbook-Detonate_URL_-_Lastline.yml]: This task does not output context data correctly:
447 sdfdsfeaf - workplan 2018-09-17 18-41-17

File: Playbooks/playbook-Detonate_File_-_Lastline.yml [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+      }
+    note: false
+  "10":
+    id: "10"
+    taskid: d8e19b0c-3dfa-41f8-86fa-2d12ff711340
+    type: condition
+    task:
+      id: d8e19b0c-3dfa-41f8-86fa-2d12ff711340
+      version: -1
+      name: Is File type supported?
```
Comment (BenJoParadise) [inline Playbooks/playbook-Detonate_File_-_Lastline.yml]: File in test playbook is of unsupported type.

File: Playbooks/playbook-Detonate_File_-_Lastline.yml (hunk 1 above)
Comment (bakatzir) [reply Playbooks/playbook-Detonate_File_-_Lastline.yml]: Fixed for all fields.

File: Playbooks/playbook-Detonate_URL_-_Lastline.yml (hunk 2 above)
Comment (bakatzir) [reply Playbooks/playbook-Detonate_URL_-_Lastline.yml]: added relevant output fields in integration

**Comment by bakatzir** [comment]
@michalgold  
Can you also review the output names and the new-command names that i've chosen?
Or we'll just schedule some time.
+according to lastline it can detonate filetypes such as xls,pdf,doc
but when I tried to do that I got an error announcing that the file type is not - How should I handle this issue?

**Comment by bakatzir** [comment]
#13399

**Comment by bakatzir** [comment]
lastline api - https://user.lastline.com/papi-doc/api/html/analysis/overview.html?highlight=upload
lastline file types - https://analysis.lastline.com/analyst-api-docs/html/overview.html

**Pull request merged**